app/core/config.py

```
@@ -0,0 +1,293 @@
+        
+        Raises:
+            ValueError: If the port number is not between 1 and 65535 (inclusive).
+        
+        Example:
+            valid_port = Settings.validate_port(8080)  # Returns 8080
+            invalid_port = Settings.validate_port(70000)  # Raises ValueError
+        
+        Generated by CodeRabbit
+        """
+        if not 1 <= v <= 65535:
+            raise ValueError("Port must be between 1 and 65535")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)